radar-upload-frontend/src/axios/file.js

```diff
@@ -0,0 +1,112 @@
+import axios from 'axios';
+import uuidv1 from 'uuid/v1';
+import { downLoadFile } from '@/helpers';
+
+export default {
+  /** response
+    {
+    "sourceTypes": [
+      {
+        "name": "Mp3Audio",
+        "contentTypes": ["application/mp3", "audio/mp3"],
+        "topics": [
+          "high_quality_mp3_audio",
+          "low_quality_mp3_audio"
+        ],
+        "timeRequired": true
+      },
+      {
+        "name": "WrittenText",
+        "contentTypes": ["text/csv", "text/json"],
+        "topics": ["written_text"],
+        "timeRequired": false
+      }
+    ]
+  }
+   */
+  getSourceTypes() {
+    return axios.get('/source-types').then(res => res.sourceTypes.map(el => ({
+      name: el.name,
+      contentTypes: el.contentTypes,
+    })));
+  },
+  // upload has 2 phases
+  // Post /records with file info to get id then
+  // PUT /records/{id}/contents/{fileName} to upload the file
```

Comment:
Currently, once you upload the client should explicitly set the status to "READY". This is not done currently, I assume @khoathanglong